recipes/freetype/all/test_package/test_package.c

```diff
@@ -69,9 +69,13 @@ main( int     argc,
   int           target_height;
   size_t        n, num_chars;
 
+  if (argv < 2) {
+    fprintf(stderr, "Usage: %s FONT\n", argv[0]);
+    return EXIT_FAILURE;
+  }
 
-  filename      = "OpenSans-Bold.ttf";
-  text          = "Bincrafters";
```

Comment:
:cry:

Reply:
we're still here https://github.com/conan-io/conan-center-index/search?q=bincrafters :D